deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -1596,6 +1598,20 @@ class ChatViewController: UITableViewController, UITableViewDropDelegate {
         navigationController?.present(alert, animated: true, completion: nil)
     }
 
+    private func showInvalidUnencryptedDialog() {
+        let alert = UIAlertController(title: String.localized("invalid_unencrypted_explanation"), message: nil, preferredStyle: .alert)
```

Comment:
Just out of curiosity:

Is there a reason you put this (long) text into the title (and not the message)?

Reply:
for this PR: just did as always :)

iirc, it otherwise rendered with a very small font (probably assuming you use title+message)